Revolution Rotation Calculator.py

```diff
@@ -333,7 +316,7 @@ def Validate(configurations):
     Time *= 0.6
 # --- Functions are layed out here --- #
 def Auto_Available(): # Will check if an auto attack is needed to be used
-    Is_Available = True
+    Is_Available = True #This variable is never used..

```

Comment:
What is the purpose of this variable?

Reply:
You're right, probably a flag I forgot to delete.